Moderation-Policy.md

```diff
@@ -1,154 +1,305 @@
+  following the above procedures including: removing Posts, Banning permanently,
+  and reporting accounts to GitHub.
+
+Note that Moderating non-Collaborator posts can often lead to retaliation or
+escalation of inappropriate behavior by the individual whose post is being
+Moderated. This is true primarily of individuals whose intent is to harass,
+disrupt or annoy individual members of the community. In such cases, it is best
+to handle the Moderation as quickly and as quietly as possible without drawing
+any further undue attention to the Post in question.
+
+### Temporary Interaction Limits
+
+The Moderation Team may, at their discretion, choose to enable GitHub's
+[Temporary Interaction Limits][] on any GitHub repository in the Node.js GitHub
+Organization.
+
+Any Collaborator may request that the Moderation Team enable the Temporary
+Interaction Limits by posting an issue to the moderation repository. If the
+Moderation Team choose not to do so, then a comment explaining why that
+decision was made should be added to the moderation repository thread.
+
+### Temporary and Indefinite Bans
+
+An Indefinite Ban is set for an unspecified period of time and may only be
+lifted for an individual through a simple majority vote of the the Moderation
+Team.
+
+A Temporary Ban is time limited, with the timeframe decided on by the Moderation
+Team at the time of issuing, depending on the severity of the issue. Recommended
+default options are 24-hour, 48-hour, and 7-day periods.
+
+## Privacy of the nodejs/moderation Repository
+
+The nodejs/moderation Repository is used to discuss the potentially sensitive
+details of any specific moderation issue. The repository is private but
+accessible to all Collaborators. The details of any issue discussed within the
+nodejs/moderation repository are expected to remain confidential and are not to
+be discussed in any public forum or social media service.
+
+Any Collaborator found to be violating the privacy of the nodejs/moderation
+repository by repeatedly sharing or discussing the details of nodejs/moderation
+issues in any public forum or social media service risks being permanently
+removed from the Node.js GitHub organization.
+
+## Moderation Team
+
+The Node.js Moderation Team is tasked with enforcement of this policy.
+
+Moderation team members are Collaborators nominated by either the TSC or
+CommComm and must be approved by *both* committees. If there are no objections
+after 72 hours, the nomination becomes automatic. If there are objections to
+a specific nomination, then a simple majority vote of each the TSC and CommComm
+members in *favor* of the nomination is required. Moderators are removed from
+the Moderation Team following the same process. Votes to remove moderators
+succeed only if a simple majority of both committees is in *favor* of removal.
+
+Once per month, the Moderation Team must provide a report of all Moderation
```

Comment:
My presumption is that would be left up to Moderation Team to determine the details for. I'd prefer not to be overly prescriptive on the specific process. It could even be something that is fully automated and updated incrementally.